sub_sensor_path: /mobile_base/events/bumper
